### PR TITLE
Report original execution time in unpack operation results

### PR DESCRIPTION
**Pull request by lptr**

### CODE REVIEW

**Comment by lptr** [comment]
Branch accept build: https://builds.gradle.org/viewQueued.html?itemId=9642143

**Comment by wolfs** [comment]
Didn't we want to add the time to `ExecuteTaskBuildOperationResult`?

**Comment by lptr** [comment]
It makes a lot more sense on the unpack operation.

File: subprojects/build-cache/src/main/java/org/gradle/caching/internal/controller/BuildCacheLoadCommand.java [hunk 1]
```diff
@@ -44,13 +44,17 @@
...
+         */
+        long getOriginalExecutionTime();
+
         /**
          * Any metadata about the loaded artifact.
          *
          * Value may not be null.
          */
         T getMetadata();
-
```
Comment (wolfs) [inline subprojects/build-cache/src/main/java/org/gradle/caching/internal/controller/BuildCacheLoadCommand.java]: Disagreement in new lines?

**Pull request merged**